### ISSUE TITLE
[Mechanics] Spellpower scaling with "when struck" items

### ISSUE THREAD

**Issue by Chenocide** [dalaranwow/dalaran-wow#4704]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

Multiple items from vanilla with the attribute "when struck in combat" or similar attributes, seems to scale with spell power. This means that the effect gets increasingly more and more powerful to the point where attackers takes several thousand points of damage. 

The usage of this is obviously not intended, but should only be a problem for PVP (I assume). 

**Expected behaviour**: Tell us what should happen instead

Said items should either scale differently or not at all. Combining all of these items on one character should not amount to the point where level 70+ characters and npcs die in just a few hits. 

**Steps to reproduce the problem**:

1. Acquire said items (linking ID's below).
2. Equipt said items.
3. Have enemy character or npc hit you.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

Have 2 old videos to prove that this has been an issue for a long time:

Video 1:
https://www.youtube.com/watch?v=eMyoPqDHjf0

Video 2: 
https://www.youtube.com/watch?v=-ycYWFWaMDE

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

I'm not sure that I know of all the items but these are what I got:

Talisman of Binding Shard - ID: 17782 (Is apparently not listed in the combat log)
Cloak of Fire - ID: 14134
Fetish of Chitinous Spikes - ID: 21488
Drillborer Disk - ID: 17066
Darkmoon Card: Maelstrom - ID: 19289
Heart of Wyrmthalak - ID: 22321
Razor Gauntlets - ID: 18326

I'm fairly certain that there are other items but I'm not aware of them.

**Include Screenshots from the issue if necessary**

Taken from a druid with me using auto attacks:
https://puu.sh/vMb7W/e61819fe0a.png

Patch 2.4.1 spell power scaling - comments from Drillborer Disk
https://puu.sh/vMbCR/94b5f89990.png

Patch 3.0.3 spell power scaling - comments from Cloak of Fire with patch nerf
https://puu.sh/vMbLk/922a2ccb04.png

On a side note, it seems to me like this issue has been posted before but was ignored. I'm gonna keep making this report until I see efforts to fix this. 

Edit* 
This should be very fast and easy to test.

**Comment by Akensai** [comment]
I'm the druid in the screenshot (Akens).

This was originally reported 2+ years ago by multiple people. The issue was mostly ignored save for Ragnaros items being fixed.

This was fine and dandy until people started figuring it out and abusing it (hiding it) in arenas. There are multiple people on the server using the neck (as it does not show in combat log) during arena.

This needs to be fixed now, not hidden. Especially with ICC on the way.

**Comment by TeridanDW** [comment]
Hello, thank you for the report @Chenocide and @Akensai !
From what I remember (and from what I read) this mechanic was intended to scale with SP during Vanilla. Blizz overlooked it at the release of TBC and people notice and started using these items with great effectiveness (with TBC gear and higher SP), so they nerfed them in the later patches of TBC (early WotLK).

I can **Confirm** the issue and will combine the list of items together with the spells they cast. I think every spells have to be fixed individually (not completely sure).

**List:**

Talisman of Binding Shard - item ID: 17782 - spell ID: 21991 [Talisman of Binding](http://www.wowhead.com/spell=21991/talisman-of-binding)
Cloak of Fire - item  ID: 14134 - spell ID: 18364 [Cloak of Fire](http://www.wowhead.com/spell=18364/cloak-of-fire)
Fetish of Chitinous Spikes - item  ID: 21488 - spell ID: 26168[Chitinous Spikes](http://www.wowhead.com/spell=26168/chitinous-spikes)
Drillborer Disk - item ID: 17066 - spell ID: 15438 [Thorns](http://www.wowhead.com/spell=15438/thorns)
Darkmoon Card: Maelstrom - item  ID: 19289 - spell ID: 23686[Lightning Strike](http://www.wowhead.com/spell=23686/lightning-strike)
Heart of Wyrmthalak - item   ID: 22321 - spell ID: 27656 [Flame Lash](http://www.wowhead.com/spell=27656/flame-lash)
Razor Gauntlets - item ID: 18326 - spell ID: 20886 [Thorns Dmg+3](http://www.wowhead.com/spell=20886/thorns-dmg-3)
Ramstein's Lightning Bolts - item ID: 13515 - spell ID: 17668 [Ramstein's Lightning Bolts](http://www.wowhead.com/spell=17668/ramsteins-lightning-bolts)
Oil of Immolation - item ID: 8956 - spell ID: 11350 [Fire Shield](http://www.wowhead.com/spell=11350/fire-shield)

Regarding:

> (Is apparently not listed in the combat log)

this seems to be intended by blizz, because in the client dbc spell file you can see for spell:

**ID - 21991 Talisman of Binding (Rank 1)**
Attributes: 0x00000140 (SPELL_ATTR0_PASSIVE, **SPELL_ATTR0_HIDE_IN_COMBAT_LOG**)



If there are more items like these (that shouldn't scale with SP) please comment below. I will also keep looking for more.

**Comment by Akensai** [comment]
@TeridanDW The Ragnaros item (not at PC to check it, I think trinket) was fixed a couple years ago with the original complaints. You may want to see what was done there.

In MaNGOS it was a formula constructed in the database.

**Comment by TeridanDW** [comment]
I couldn't find any report relevent to this issue. Do you know which one is it?

**Comment by Akensai** [comment]
It was not submitted to github as nobody wanted it known what items would
do it (or that it was bugged). I believe it was Loth that took care of that.

On May 11, 2017 4:54 PM, "Teridan" <notifications@github.com> wrote:

I couldn't find any report relevent to this issue. Do you know which one is
it?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/dalaranwow/dalaran-wow/issues/4704#issuecomment-300914106>,
or mute the thread

**Comment by Akensai** [comment]
@TeridanDW 
I've found a list of all the items that use the same attribute:-
http://wowwiki.wikia.com/wiki/Damage_shield

Based on this (gave the list to Cheno) quite a few where missed and I suspect they all scale accordingly due to the same functions.

**Comment by Edzero** [comment]
https://github.com/dalaranwow/dalaran-wow/issues/3938

**Comment by Jiaku** [comment]
Since this official and everybody knows about, till it fixed everybody abuses it and it become unplayable

**Comment by Chenocide** [comment]
People were exploiting it before I posted this. Dal-wow is like the special kid in class when it comes to actually getting fixes pushed to live but I hope this can motivate them to prioritize it higher. If getting this fixed means that we need 50 people using this, then I'll take that over this not getting fixed at all.

**Comment by TeridanDW** [comment]
Hello, a fix for this issue was produced 7 days ago by @robinsch (Thank you), it is just a matter of time until it is applied to live. Leaving the issue open as a developer has to close it.

P.S. Please avoid posting any comments that aren't related to the issue itself.

**Comment by Rushor** [comment]
Core/Item: Stop damage shield auras from scaling with modifiers.

Fixes: https://github.com/dalaranwow/dalaran-wow/issues/4704
* by @robinsch 
* commit: e95c245a15525e1af224d19fb9f9a0dfa29ce964
* commit: 9de3832360d8bd5432152d8088e10dfa2720c4dc (from https://github.com/dalaranwow/dalaran-wow/issues/3938)

The issue was fixed by a developer and will be tested on the Public Test Realm in the future (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm in the future.
This means that the issue exists in game at the moment i wrote this message.
**Only once the issue was marked with "Fixed - On Live Server", you can find the fix on your game server.**
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by Lothloryen** [comment]
Hello there,

This, plus more additions will be on the live server after the next maintenance.

**Comment by Akensai** [comment]
> Hello there,
> 
> This, plus more additions will be on the live server after the next maintenance.

A little late now. 👍
